features/local-storage.html

```diff
@@ -0,0 +1,74 @@
+
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Local Storage & Cookies storing</title>
+</head>
+<body>
+    <h1>Local Storage & Cookies storing</h1>
+    <p></p>
+    <h3>Initial values</h3>
+    <div id='initial'></div>
+    <p>
+      <button id='start'>Start test</button>
+      <button id='cleanup'>Cleanup data</button>
+    </p>
+
+    <h3>Current values</h3>
+    <div id='tests'>
+    </div>
+
+    <script>
+      const startButton = document.querySelector('#start');
+      const cleanupButton = document.querySelector('#cleanup');
+      const initialDiv = document.querySelector('#initial');
+      const testsDiv = document.querySelector('#tests');
+
+      startButton.addEventListener('click', () => startIncrementing());
+      cleanupButton.addEventListener('click', () => cleanup());
+
+      initialDiv.innerHTML = countInfoAsHtml();
+
+      function startIncrementing () {
+          setInterval(() => {
```

Comment:
if you'd like to stop this on "cleanup" button click:

```js
let someVariable = setInterval(…)

…

clearInterval(someVariable)
```

Reply:
I don't need to, but thanks for the tip :)